astroquery/exoplanets/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from .exoplanet_archive import *
+from .exoplanets_org import *
+from .planet_params import *
```

Comment:
Could you use explicit imports here?  Especially since there is no `core.py`, it is helpful to be able to look at `__init__.py` to determine what's in the exoplanets namespace.